functions/src/call-cloud-function.ts

```diff
@@ -1,17 +0,0 @@
-import * as admin from 'firebase-admin'
```

Comment:
I guess we can delete this, since there's probably a better alternative to calling cloud functions from cloud functions (i.e. factor our a common function and call it in both, or increase memory and timeout limits)